src/lib/actions/input.ts

```diff
@@ -3,7 +3,7 @@ import type { Action } from 'svelte/action';
 /**
  * Auto focus node when rendered.  Useful for inputs
  */
-export const autoFocus: Action<HTMLElement, { delay?: number }> = (node, options?) => {
```

Comment:
I assume using the Svelte provided `Action<>` interface is causing some of your problems?

Reply:
Yeah, *something* about the way it's being interpreted with the current syntax causes it to assume misinformation about the method signature and I couldn't seem to figure out what I was doing wrong, but when I changed the style of the function declaration, the issue just *magically* disappeared, lol.

Reply:
I feel like I've been getting more actions complaining in typescript lately... not sure if something changed in Svelte 4, Svelte UX, or I've just never noticed them as much.  If it works for you, I'm all for it :)

Reply:
It changed in Svelte 4. They made it so that it now requires a generic argument for the arguments you pass to the action when it's used.